controllers/department.js

```diff
@@ -1,31 +1,61 @@
-const Department = require("../models/department");
+const DepartmentService = require("../services/department");
+const EmployeeService = require("../services/employee");
 
-class departmentController {
+class DepartmentController {
+  async createNewDepartment(request, response) {
+    try {
+      const department = await DepartmentService.createNewDepartment(request.body, request.files.picture);
+      return response.status(200).json({ department });
+    } catch (e) {
+      console.log(e.message);
```

Comment:
may be .log should be changed to .error and both output and res.status.... should be the separate function. in that case it would be smth like 

catch(e) {
   return errorHandler(res, e, "Custom message");
}